test/k8sT/Verifier.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8sTest
+
+import (
+	"fmt"
+
+	. "github.com/cilium/cilium/test/ginkgo-ext"
+	"github.com/cilium/cilium/test/helpers"
+
+	. "github.com/onsi/gomega"
+)
+
+const (
+	script      = "bpf/verifier-test.sh"
+	podName     = "test-verifier"
+	podManifest = "test-verifier.yaml"
+)
+
+var _ = Describe("K8sVerifier", func() {
```

Comment:
I think it would be nice to describe (hehe :slightly_smiling_face:) what this test is doing and how it is different than other K8s tests. I feel it's important to call out that this is not a K8s test per se, but rather we are utilizing the K8s test infrastructure to run BPF verifier tests. I see that this is explained in the commit msg, but I think it would be good to also include in the src as well.

Reply:
Makes sense. I added a comment at the top to explain the test's objective and its particularity.